### ISSUE TITLE
Remote control screen closed if we open UAC application on remote machine.

### ISSUE THREAD

**Issue by dharmikmshah** [Jay-Rad/Remotely#389]
**Describe the bug**
Not able to access the machine if we start any application that requires UAC (User Account Control).

**To Reproduce**
1. Connect the client 
2. On Client side open any of the application that requires UAC.
3. As soon as we start the application on remote side complete screen will be blank, no way to get out from that.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser [chrome]
 
![1](https://user-images.githubusercontent.com/76530669/140759919-53947f8b-f2c5-46f9-8885-0ef5ed58aa45.png)
![2](https://user-images.githubusercontent.com/76530669/140759927-ca1bbdc3-c4ab-4fa4-b220-ae9bb91a735e.JPG)

**Comment by enclustra-admin** [comment]
This is the same if you open a dialog which needs UAC. E.g. "Advanced system setting". You see the dialog, the mouse pointer changes to a hand with fingers, but can't click anything until someone with direct access to the PC closes it.
This makes it less usefull for ad hoc helping.

@dharmikmshah
Hmm, this looks like a feature request. 😉 Because under the settings button is a "Elevate to Admin" which also (of course) leeds to the same problem. Out of curiosity (didn't try it myself) Did you already try to install it as a service?

**Comment by mziminski616** [comment]
A year later and we still don't have this feature 😞 . I just put in a lengthy feature request which explains exactly what we're looking for. Both TeamViewer and Connectwise Control (of course both paid options) implement it beautifully!

**Comment by hooligans44** [comment]
Yes this User account control (UAC)  feature need to be add on portable clients downloadable plug .
